proxy/src/main/java/net/md_5/bungee/UserConnection.java

```diff
@@ -548,6 +550,50 @@ public Locale getLocale()
     }
 
     @Override
+    public byte getViewDistance()
+    {
+        return ( settings != null ) ? settings.getViewDistance() : 10;
```

Comment:
Use + document a sentinel value such as -1.
10 is a valid view distance, which could get confusing.